app/src/main/java/com/kickstarter/ui/views/compose/checkout/PaymentSchedule.kt

```diff
@@ -164,14 +182,57 @@ fun PaymentRow(paymentIncrement: PaymentIncrement, ksCurrency: KSCurrency?, proj
             )
             StatusBadge(paymentIncrement.state)
         }
+
         Text(
             modifier = Modifier.testTag(PaymentScheduleTestTags.AMOUNT_TEXT.name),
-            text = formattedAmount,
+            text = paymentIncrementStyledCurrency(paymentIncrement, ksCurrency),
             style = typography.title3
         )
     }
 }
 
+@Composable
+private fun paymentIncrementStyledCurrency(
+    paymentIncrement: PaymentIncrement,
+    ksCurrency: KSCurrency?
+): AnnotatedString {
+    val country = Country.findByCurrencyCode(paymentIncrement.amount().currencyCode ?: "")
+    val currencySymbol = country?.let { ksCurrency?.getCurrencySymbol(it, false) } ?: ""
+
+    val currencyToFormat = "${currencySymbol.trimAllWhitespace()} ${paymentIncrement.amount().amountAsFloat}"
```

Comment:
used `paymentIncrement.amount().amountAsFloat` instead of `formattedAmountWithCode` to keep consistency with the rest of manage pledge screen. 
`"USD $99.75"` the 'D' does not appear in any other amount displayed in the screen